.github/workflows/build-windows.yml

```diff
@@ -1,14 +1,38 @@
 name: Build for Windows
 
-on: workflow_dispatch
+on:
+  workflow_dispatch:
+    inputs:
+      edgar_renderer_ref:
+        description: 'Arelle/EdgarRenderer branch, tag or SHA to checkout (blank for default)'
+        required: false
+        type: string
+      ixbrl_viewer_ref:
+        description: 'Workiva/ixbrl-viewer branch, tag or SHA to checkout (blank for default)'
+        required: false
+        type: string
+      node_version:
+        default: lts/*
+        description: 'Node.js version to use'
+        required: true
+        type: string
+      python_version:
+        default: '3.9'
```

Comment:
The windows workflow [has a few](https://github.com/Arelle/Arelle/blob/211fb1ba912058f8401c09a93a2cc804d79ae92a/.github/workflows/build-windows.yml#L76) references [to the python version](https://github.com/Arelle/Arelle/blob/211fb1ba912058f8401c09a93a2cc804d79ae92a/.github/workflows/build-windows.yml#L79). Can you update those to use the matrix value?

There's additional work to make the windows build scripts actually work with other python versions, but you don't need to worry about getting that to work here.

Reply:
Just noticed an issue with using patch versions. The build directory gets named based on the <MAJOR>.<MINOR> version without the patch #. Is that what you're referring to?

Reply:
There's also some stuff with the installer. That doesn't need to be addressed here. The main goal of this is to get the builds working with alternative plugin branches.